nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -43,20 +44,33 @@
 |               | `rght`        |
 |               | `tree_id`     |
 
+### Replaced Models
+
+The `ipam.Role`, `dcim.RackRole`, and `dcim.DeviceRole` models have been removed and replaced by a single `extras.Role` model. This means that any references to the removed models in the code now use the `extras.Role` model instead.
+
+| Removed Model     | Replaced With  |
+|-------------------|----------------|
+| `dcim.DeviceRole` | `extras.Role`  |
+| `dcim.RackRole`   | `extras.Role`  |
+| `extras.Role`     | `extras.Role`  |
+| `ipam.Role`       | `extras.Role`  |
+
 ## GraphQL and REST API Changes
 
 ### API Behavior Changes
 
-| Model       | Field          | Changes                                                                                                 |
-|-------------|----------------|---------------------------------------------------------------------------------------------------------|
-| RackGroup   | `rack_count`   | Now only counts Racks directly belonging to this RackGroup, not those belonging to its descendants.     |
-| Region      | `site_count`   | Now only counts Sites directly belonging to this Region, not those belonging to its descendants.        |
-| TenantGroup | `tenant_count` | Now only counts Tenants directly belonging to this TenantGroup, not those belonging to its descendants. |
+| Model       | Field          | Changes                                                                                                  |
+|-------------|----------------|----------------------------------------------------------------------------------------------------------|
+| Ipaddress   | `role`         | `/ipam/ip-addresses/` endpoint now uses role nested serializer for the role field, rather than a string. |
```

Comment:
```suggestion
| IPAddress   | `role`         | `/ipam/ip-addresses/` endpoint now uses role nested serializer for the role field, rather than a string. |
```